src/Features/Core/Portable/Completion/CompletionOptionsProvider.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System.Collections.Generic;
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@dpoeschl i've been a bit lax lately and haven't run it.  My plan was to pick it back up once our update 1 changes started to settle down in stabilization.  Running it now is likely to make it harder to flow stabilization -> master due to merge conflicts.